resources/l10n/tr_TR/LC_MESSAGES/loot.po

```diff
@@ -0,0 +1,1446 @@
+msgstr "Ana liste güncellemesi gerekli değildi."
+
+#: src/gui/qt/main_window.cpp:2594
+#, c++-format
+msgid "Masterlist updated to revision {0}."
+msgstr "Ana liste {0} revizyonuna güncellendi."
+
+#: src/gui/qt/main_window.cpp:2658
+msgid "No masterlist updates were necessary."
+msgstr "Ana liste güncellemeleri gerekli değildi."
+
+#: src/gui/qt/main_window.cpp:2679
+msgid ""
+"Masterlists updated for the following games:\n"
+"\n"
+msgstr ""
+"Aşağıdaki oyunlar için ana listeler güncellendi:\n"
+"\n"
+
+#: src/gui/qt/main_window.cpp:2741
+#, c++-format
+msgid "A [new release]({0}) of LOOT is available."
+msgstr "LOOT'un [yeni bir sürümü]({0}) mevcut."
+
+#: src/gui/qt/main_window.cpp:2758
+msgid ""
+"Failed to check for LOOT updates! You can check your LOOTDebugLog.txt (you "
+"can get to it through the main menu) for more information."
+msgstr ""
+"LOOT güncellemeleri kontrol edilemedi! Daha fazla bilgi için LOOTDebugLog."
+"txt dosyanızı (ana menüden ulaşabilirsiniz) kontrol edebilirsiniz."
+
+#: src/gui/qt/plugin_card.cpp:156
+msgid "Source:"
+msgstr "Kaynak:"
+
+#: src/gui/qt/plugin_card.cpp:157
+msgid "Sources:"
+msgstr "Kaynaklar:"
+
+#: src/gui/qt/plugin_card.cpp:173
+#, c++-format
+msgid "Verified clean by {0}"
+msgstr "Temiz olduğu {0} tarafından onaylandı."
+
+#: src/gui/qt/plugin_card.cpp:287
+msgid "Active Plugin"
+msgstr "Aktif Eklenti"
+
+#: src/gui/qt/plugin_card.cpp:288
+msgid "Master Plugin"
+msgstr "Ana Eklenti"
+
+#: src/gui/qt/plugin_card.cpp:289
+msgid "Light Plugin"
+msgstr "Az Etkili Eklenti"
+
+#: src/gui/qt/plugin_card.cpp:290
+msgid "Empty Plugin"
+msgstr "Boş Eklenti"
+
+#: src/gui/qt/plugin_card.cpp:291
+msgid "Loads Archive"
+msgstr "Arşivi Yükler"
+
+#: src/gui/qt/plugin_card.cpp:292 src/gui/qt/plugin_item_model.cpp:128
+msgid "Has User Metadata"
+msgstr "Kullanıcı Meta Verisine Sahip"
+
+#: src/gui/qt/plugin_card.cpp:294
+#: src/gui/qt/plugin_editor/plugin_editor_widget.cpp:192
+msgid "Bash Tags"
+msgstr "Bash Etiketleri"
+
+#: src/gui/qt/plugin_card.cpp:295
+msgid "Current"
+msgstr "Geçerli"
+
+#: src/gui/qt/plugin_card.cpp:296 src/gui/qt/plugin_editor/table_tabs.cpp:524
+msgid "Add"
+msgstr "Ekle"
+
+#: src/gui/qt/plugin_card.cpp:297 src/gui/qt/plugin_editor/table_tabs.cpp:524
+msgid "Remove"
+msgstr "Kaldır"
+
+#: src/gui/qt/plugin_item_model.cpp:126
+msgid "Editor Is Open"
+msgstr "Düzenleyici Açık"
+
+#: src/gui/qt/plugin_item_model.cpp:177
+msgid "Position"
+msgstr "Pozisyon"
+
+#: src/gui/qt/plugin_item_model.cpp:179
+msgid "Index"
+msgstr "Dizin"
+
+#: src/gui/qt/plugin_item_model.cpp:181
+msgid "Plugin Name"
+msgstr "Eklenti Adı"
+
+#: src/gui/qt/search_dialog.cpp:109
+msgid "Search Cards"
+msgstr "Kartları Ara"
+
+#: src/gui/qt/search_dialog.cpp:111
+msgid "Search cards"
+msgstr "Kartları ara"
+
+#: src/gui/qt/search_dialog.cpp:115
+msgid ""
+"If checked, interprets the search text as a Perl-like regular expression."
+msgstr "İşaretlenirse, içerik filtresi metnini düzenli ifade olarak yorumlar."
```

Comment:
Since `Perl` is a programming language, it should be mentioned within the translation as well. Otherwise the intent of the original string wouldn't be represented within the translation properly.